recipes/install.rb

```diff
@@ -68,6 +68,35 @@
 
 directory '/etc/mesos-chef'
 
+case node['mesos']['init']
+when 'systemd'
+  node.default['mesos']['master']['env']['ULIMIT'].gsub(/[^0-9]/, '')
```

Comment:
Exactly the reason why it should be in the attributes file. One look at it and one should know what value shall be applied.